public_html/index.html

```diff
@@ -11,8 +11,8 @@
     <link rel="icon" type="image/png" sizes="196x196" href="assets/img/favicon-196x196.png">
     <link rel="shortcut icon" href="/favicon.ico">
     <link rel="apple-touch-icon" href="assets/img/apple-touch-icon.png">
-    <script src="https://use.typekit.net/glx7slj.js"></script>
-    <script>try{Typekit.load({ async: true });}catch(e){}</script>
+    <script src="https://use.typekit.net/glx7slj.js" defer></script>
```

Comment:
this (and the verbose `html[wf-*]` styles above) make sure that loading the web-fonts from TypeKit doesn't block rendering of the page. it previously did. big perf improvement! 👍